sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java

```diff
@@ -58,4 +58,53 @@ public Contact buildGeneratedContact() {
         .descriptionOfHowContactTookPlace("Automated test dummy description")
         .build();
   }
+
+  public Contact buildEditContact() {
+    return Contact.builder()
+        .classification("CONFIRMED CONTACT")
+        .multiDay("Multi-day contact")
+        .dateOfFirstContact(LocalDate.now().minusDays(3))
+        .dateOfLastContact(LocalDate.now().minusDays(2))
+        .diseaseOfSourceCase("Measles")
+        .externalId(UUID.randomUUID().toString())
+        .externalToken(UUID.randomUUID().toString().substring(0, 8))
+        .reportDate(LocalDate.now())
+        .reportingDistrict("District11")
+        .responsibleRegion("Region1")
+        .responsibleDistrict("District12")
+        .responsibleCommunity("")
```

Comment:
I added a responsible Community, in order to check a valid filled value, please update the empty string.